### PR TITLE
BETA: Register akashr.is-a.dev

### PR DESCRIPTION
**Pull request by CruxCoder7**
Added `akashr.is-a.dev` using the [dashboard](https://manage.is-a.dev).

### CODE REVIEW

File: domains/akashr.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "CruxCoder7",
+        "email": "akashrangesh07@gmail.com"
+    },
+    "record": {
+        "CNAME": "akashr.vercel.app"
```
Comment (SX-9) [inline domains/akashr.json:7]: we dont support vercel

File: domains/akash.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "CruxCoder7",
+        "email": "akashrangesh07@gmail.com"
+    },
+    "record": {
+        "CNAME": "akash"
```
Comment (SX-9) [inline domains/akash.json:7]: invalid value (no tld)

**Review by satr14washere — CHANGES_REQUESTED** [review]

File: domains/akashr.json (hunk 1 above)
Comment (CruxCoder7) [reply domains/akashr.json:7]: Can you please tell me the hosting providers that you support?

Comment (SX-9) [reply domains/akashr.json:7]: we support almost every platform except vercel and netlify

**Pull request closed without merging**